### PR TITLE
Feature - Changing opencb version to 1.2.1

### PR DESCRIPTION
**Pull request by priesgo**
Please @greglever

### CODE REVIEW

File: protocols/util/generate_mock_objects.py [hunk 1]
```diff
@@ -1712,6 +1713,7 @@ def get_valid_api_0_1_0_SNAPSHOT():
 
     new_api.status = system_0_1_0_SNAPSHOT.Status.OK
     new_api.type = system_0_1_0_SNAPSHOT.APIType.REST
+    new_api.url = ['']
```
Comment (greglever) [inline protocols/util/generate_mock_objects.py]: This line needs to be deleted, then the tests will pass

File: protocols/util/generate_mock_objects.py [hunk 2]
```diff
@@ -1701,6 +1701,7 @@ def get_valid_data_store_0_1_0_SNAPSHOT():
     new_datastore = object_type()
 
     new_datastore.status = system_0_1_0_SNAPSHOT.Status.OK
+    new_datastore.url = ['']
```
Comment (greglever) [inline protocols/util/generate_mock_objects.py]: this line needs to be deleted, then the tests will pass

**Comment by priesgo** [comment]
Works here... don't know why..
```
priesgo@londoner2:~/src/gel/GelReportModels$ git pull origin change-opencb-version
From github.com:genomicsengland/GelReportModels
 * branch            change-opencb-version -> FETCH_HEAD
Already up-to-date.
priesgo@londoner2:~/src/gel/GelReportModels$ python -m unittest discover
..................................................................
----------------------------------------------------------------------
Ran 66 tests in 1.739s

OK
```

**Comment by greglever** [comment]
```
$ grep import schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/* | grep 1.3.0-SNAPSHOT
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/EvidenceSet.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/variant.avdl";
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/EvidenceSet.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/evidence.avdl";
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/ObservedVariant.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/variant.avdl";
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/ObservedVariant.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/evidence.avdl";
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/ReportedVariant.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/evidence.avdl";
schemas/IDLs/org.gel.models.cva.avro/0.4.0-SNAPSHOT/Variant.avdl:    import idl "../../org.opencb.biodata.models/1.3.0-SNAPSHOT/variant.avdl";
```

**Comment by greglever** [comment]
`grep` is your friend

**Comment by greglever** [comment]
building...

**Comment by greglever** [comment]
Successful build and tests passing ! :smile:

**Pull request merged**